packages/dds/tree/src/shared-tree-core/editManager.ts

```diff
@@ -429,3 +547,33 @@ function getPathFromBase<TChange>(
 	);
 	return path;
 }
+
+function isTrunkCommit<TChange>(commit?: GraphCommit<TChange>): commit is TrunkCommit<TChange> {
+	return commit !== undefined && (commit as TrunkCommit<TChange>).sequenceNumber !== undefined;
+}
+
+function mintTrunkCommit<TChange>(
```

Comment:
What happens if a SequencedCommit is passed to the first overload?

Reply:
It doesn't complain; the sequence number in the commit will be ignored.